web/_js/atlas.js

```diff
@@ -512,6 +512,7 @@ var atlas = [
 { "id": 462, "submitted_by": "Beegrene", "name": "Octavia", "description": "Octavia is a minor character from My Little Pony Friendship is Magic. She is often seen playing various stringed instruments. The '2SET' in her mane refers to TwoSet Violin, a comedy duo.", "website": "", "subreddit": "/r/mylittlepony", "center": [ 1343.5, 1985.5 ], "path": [ [ 1316.5, 1973.5 ], [ 1370.5, 1973.5 ], [ 1370.5, 1997.5 ], [ 1316.5, 1997.5 ], [ 1316.5, 1973.5 ] ] },
 { "id": 463, "submitted_by": "xXLInkster17Xx", "name": "Little Big Planet","description": "r/LittleBigPlanet was created by several dedicated members of the Little Big Planet Union, it includes the well-known Little Big Planet tree, a sackboy face to the right and the r/LittleBigPlanet sub-reddit text below the tree which used to read \"My Little Big Planet\" but was changed due to pixel limitations.", "website": "https://littlebigplanet.com/", "subreddit": "r/LittleBigPlanet", "center": [ 1046.5, 385.5 ], "path": [ [ 1064.5, 400.5 ], [ 1033.5, 400.5 ], [ 1033.5, 366.5 ], [ 1051.5, 366.5 ], [ 1051.5, 373.5 ], [ 1062.5, 373.5 ], [ 1062.5, 379.5 ], [ 1051.5, 379.5 ], [ 1051.5, 387.5 ], [ 1064.5, 387.5 ], [ 1064.5, 400.5 ] ] },
 { "id": 464, "submitted_by": "falkihr", "name": "Jeb (Kerbal Space Program)", "description": "Jebediah Kerman, the protagonist of the planetary popular Kerbal Space Program! Below Jeb, a representation of the Kerbol System. From left to right: Kerbol (Sun), Moho (Mercury), Eve (Venus), Kerbin (Earth), Duna (Mars), Jool (Jupiter), Eeloo (Pluto). Between Duna (red) and Jool (green) there's a suspicious void area… was something supposed to be there? Nah…", "website": "www.kerbalspaceprogram.com", "subreddit": "r/KerbalSpaceProgram", "center": [ 1503.5, 410.5 ], "path": [ [ 1493.5, 427.5 ], [ 1493.5, 415.5 ], [ 1495.5, 415.5 ], [ 1495.5, 397.5 ], [ 1496.5, 397.5 ], [ 1496.5, 394.5 ], [ 1497.5, 394.5 ], [ 1497.5, 393.5 ], [ 1498.5, 393.5 ], [ 1498.5, 392.5 ], [ 1500.5, 392.5 ], [ 1500.5, 391.5 ], [ 1507.5, 391.5 ], [ 1507.5, 392.5 ], [ 1509.5, 392.5 ], [ 1509.5, 393.5 ], [ 1510.5, 393.5], [ 1510.5, 394.5 ], [ 1511.5, 394.5 ], [ 1511.5, 397.5 ], [ 1512.5, 397.5 ], [ 1512.5, 427.5] ] },
+{ "id": 465, "submitted_by": "Maxi_der_Zocker", "name": "Trackmania", "description": "The logo of game released in the Year 2020 of the arcade racing game series Trackmania.", "website": "www.trackmania.com", "subreddit": "r/Trackmania", "center": [ 1596.5 , 521.5 ], "path": [ [ 1591.5, 499.5 ], [ 1600.5, 499.5 ], [ 1600.5, 499.5 ], [ 1600.5, 543.5 ], [ 1591.5,543.5 ], [ 1591.5, 499.5 ] ] },
```

Comment:
```suggestion
{ "id": 465, "submitted_by": "Maxi_der_Zocker", "name": "Trackmania", "description": "The logo of game released in the Year 2020 of the arcade racing game series Trackmania.", "website": "https://www.trackmania.com/", "subreddit": "r/Trackmania", "center": [ 1596.5 , 521.5 ], "path": [ [ 1591.5, 499.5 ], [ 1600.5, 499.5 ], [ 1600.5, 499.5 ], [ 1600.5, 543.5 ], [ 1591.5,543.5 ], [ 1591.5, 499.5 ] ] },
```
The link doesn't work, could you try this so it hopefully works?